src/main/java/seedu/address/storage/volunteer/JsonSerializableVolunteer.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.storage.volunteer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.FriendlyLink;
+import seedu.address.model.ReadOnlyVolunteer;
+import seedu.address.model.person.Volunteer;
+import seedu.address.storage.JsonAdaptedPerson;
+
+/**
+ * An Immutable Volunteer that is serializable to JSON format.
+ */
+@JsonRootName(value = "volunteer")
+public class JsonSerializableVolunteer {
+    public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
```

Comment:
This constant seems to be repeated accrosee multiple classes, perhaps change it to be more specific?
```suggestion
    public static final String MESSAGE_DUPLICATE_VOLUNTEER = "Persons list contains duplicate volunteer(s).";
```

Reply:
Yup my bad